src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```

Comment:
please remove this file
scaffolded code

Reply:
Removed